ChangeLog.d/build_with_only_montgomery_curves.txt

```diff
@@ -0,0 +1,6 @@
+Bugfix
```

Comment:
Note on the commit message: github does't handle lists, so you want `fix #941, fix #1412, fix #1147, fix #2017` here or in the PR description.

Reply:
Note: the relevant magic words are already in the PR description, so nothing to do here.